### DISCUSSION TITLE
Heavy CPU load on graphql calls

### DISCUSSION THREAD

**Discussion by joaovpmamede** [WebDevStudios/nextjs-wordpress-starter#968] [Q&A]
First of all, sorry if this is not suited here (on this section or even on this repo) but I guess I'm having a hard time figuring what the heck is happening with my project and it could be that it's happening with others ( [here](https://discourse.roots.io/t/cpu-on-server-maxes-out-on-almost-every-request/22462) and [here](https://github.com/wp-graphql/wp-graphql/issues/1303#issuecomment-833550277)) too.

So basically everyt ime I build my nextjs project it takes a lot of time and by going to the server and running `top` I've found that `php-fpm` is using a lot of CPU.
Not only that but just a simple Postman call to graphql will have the same effect.

I've already tried different "backends"/stacks ([trellis](https://roots.io/trellis/) on Digital Ocean, trellis on AWS Lightsail and a simple app container with Wordpress on Lightsail as well) and I get the same issue all the time.

Adding a Cache won't help here and in fact I'm already using Redis with [wp-graphql-cache](https://github.com/valu-digital/wp-graphql-cache).

Is this something that happens/have happened to you already?

**Comment by sloyer** [comment]
We have noticed high usage on builds as well.

**Reply by joaovpmamede** [reply]
Interesting.
Do you also noticed when doing a request to a page that wasn't generated during the build or even when doing a request through postman or any other API testing platform?
